amethyst_ui/src/event.rs

```diff
@@ -48,6 +48,10 @@ pub enum UiEventType {
         /// The entity on which the dragged object was dropped.
         dropped_on: Entity,
     },
+    /// When the value of an `Editable` Ui element has changed.
+    Change,
```

Comment:
Could it be of benefit to provide a delta or the full value with the event? Just a thought, might save the user the trouble of having to read from the field and instead just depend on the events. On the other hand, it's easy enough to implement for the user, and might be unwanted overhead that he then can't turn off.

Reply:
Impossible because you can't know the value type.

Reply:
Ok, fair